### ISSUE TITLE
Questions to better understand evaluation metrics for PPOCR recognition tasks

### ISSUE THREAD

**Issue by pyramid20002000** [PaddlePaddle/PaddleOCR#8424]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

Dear developpers,

Thank you for providing such a strong toolkit for ocr related tasks.

I used https://paddleocr.bj.bcebos.com/PP-OCRv3/chinese/ch_PP-OCRv3_det_distill_train.tar as pre-trained models.
I succeeded to train it on my dataset for OCR recognition.
Then I used tools/eval.py to evaluate the results.

Here are a few examples of evaluation:
![image](https://user-images.githubusercontent.com/109776496/203537721-23b2deec-cf83-41c2-a1cd-da8deffc7a57.png)
![image](https://user-images.githubusercontent.com/109776496/203537748-c36fd966-5445-4270-b009-dc7b433760dd.png)
![image](https://user-images.githubusercontent.com/109776496/203537780-d7cc1b8c-8b9f-4de7-8b34-2336b1755fe0.png)

Now I have some questions about these outputs:
1：what's difference between "metric in ckpt" and "metric eval"? 
Are they both for validation set?  Or one for training set and the other for validation set?
I assume that ckpt means checkpoint? Then what is a "metric in ckpt"?

2：why "best epoch" can be smaller than "start epoch"?? 

3:  How is "acc" calculated?  the number of correct recognized images / the number of all validation images ?
How is "correct recognized" defined here? Is there any document for this?

4: Similar questions for "edit distance" and "normalized edit distance".
As you may notice that various methods exist for both edit distance calculation and normalization.
Is there any document to help us better understand the outputs?

5: when I run evaluation for pre-trained weights in ch_PP-OCRv3_det_distill_train.tar.
Its score is even better than models obtained by finetuning.
(see the third images above, the one without "metric in ckpt") 
Is it normal?

**Comment by pyramid20002000** [comment]
@an1018 
您好，这个evaluation metrics没有人知道详细的计算方法吗？
提问一周没有答复感到不可思议啊。因为这里面没有技术问题，
只是一个单纯的名词解释和确认......

**Comment by an1018** [comment]
这里有指标介绍：https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/doc/doc_ch/training.md#23-%E8%AF%84%E4%BC%B0%E6%8C%87%E6%A0%87
outputs输出日志信息介绍：https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/doc/doc_ch/recognition.md#21-%E5%90%AF%E5%8A%A8%E8%AE%AD%E7%BB%83

metric in ckpt表示加载模型的指标
metric eval表示使用加载模型在当前数据集上的评估指标
acc:字符识别准确率，具体代码可以参考ppocr/metrics/rec_metric.py，也有normalized edit distance计算代码，
finetune指标低：可以看一下确认字典txt是否有需要更改、超参数调节等，可以参考下微调文档：https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.6/doc/doc_ch/finetune.md

**Comment by pyramid20002000** [comment]
感谢
